js-library/scripts/package-next

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eux
+
+npm ci
+
+: Package first the library
+npm run build
+
+: Then update the version to SEMANTIC_VERSION-next-DATE e.g. 0.0.1-next-2022-08-05T15:02
```

Comment:
The code uses `slice(0, 10)`, so the actual version would be `0.0.1-next-2022-08-05` only, right?

Reply:
Correct